package.json

```diff
@@ -1,12 +1,15 @@
 {
   "name": "gae-scaffold",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "devDependencies": {
     "grunt": "~0.4.5",
     "grunt-appengine": "~0.1.5",
     "grunt-contrib-clean": "~0.5.0",
     "grunt-contrib-copy": "~0.5.0",
     "grunt-closure-soy": "~0.2.0",
-    "grunt-closure-tools": "~0.9.6"
+    "grunt-closure-tools": "~0.9.6",
+    "google-closure-library": "*",
+    "google-closure-compiler": "*",
+    "google-closure-templates": "*"
```

Comment:
Can we specify >= some version for the closure dependencies here?  (if it makes sense, feel free to start a thread w/internal maintainers and I).  If nothing else, at least >= or ~ the versions that are published today?

My concern is that also for closure-templates the minimum library and compiler versions should be compatible with the templates so it may be worth a bit of additional discussion.